tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/net/http/resiliency/CircuitBreakerTest.java

```diff
@@ -75,7 +76,7 @@ public void testCircuitBreaker() {
         Assert.assertEquals(returnVals.length, 2);
 
         BRefValueArray responses = (BRefValueArray) returnVals[0];
-        BRefValueArray errs = (BRefValueArray) returnVals[1];
+        BRefValueArray errorsArray = (BRefValueArray) returnVals[1];
```

Comment:
I'd just name this variable "errors". Feel like naming is not consistent with the previous variable :)